central/processlisteningonport/store/store.go

```diff
@@ -8,6 +8,7 @@ import (
 )
 
 // Store provides storage functionality.
+//
```

Comment:
Was this intended to go into this PR?

Reply:
I missed this in #4648 `gofmt` has changed in 1.19